README.md

```diff
@@ -34,6 +34,29 @@ make test
 See [`CONTRIBUTING.md`](CONTRIBUTING.md) for details on how to contribute to
 this project.
 
+The code in this project is formatted with `clang-format(1)`, and our CI builds
```

Comment:
Please reflow this paragraph so it matches its rendering in Markdown. `gq}` in Vim, `M-q` in [Emacs](https://www.gnu.org/software/emacs/manual/html_node/emacs/Fill-Commands.html) (I think).

Reply:
Done.